test/library/standard/Random/ferguson/fillrandom-bounded.chpl

```diff
@@ -168,6 +168,13 @@ proc testitb(seed: int, n: int, min: uint, max: uint) {
     B[i] = x;
   }
   checkMatch(A, B);
+
+  if verbose then writeln("testitb ", seed, " ", n, " ", min, " ", max, " E");
+  A=0;
+  B=0;
+  fillRandom(A, min, max, seed);
+  fillRandom(B, min, max, seed);
+  checkMatch(A,B);
```

Comment:
I might add something similar to this as a new, standalone test rather than modifying an existing one.

I would recommend adding the new test in `test/library/standard/Random/`. Note that you will have to add a `.good` file for your test as well (see https://github.com/chapel-lang/chapel/blob/main/doc/rst/developer/bestPractices/TestSystem.rst for more details on adding a new test and the test system in general)

Reply:
(also, you don't have to have all the verbose stuff and all the extra stuff that is in this test, I think just having a test that fills two random arrays with a seed and then makes sure they are equal is good enough)